fluss-common/src/main/java/com/alibaba/fluss/memory/AbstractPagedInputView.java

```diff
@@ -0,0 +1,545 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.alibaba.fluss.memory;
+
+import java.io.EOFException;
+import java.io.IOException;
+import java.io.UTFDataFormatException;
+
+/**
+ * The base class for all input views that are backed by multiple memory pages. This base class
+ * contains all decoding methods to read data from a page and detect page boundary crossing. The
+ * concrete sub classes must implement the methods to provide the next memory page once the boundary
+ * is crossed.
+ */
+public abstract class AbstractPagedInputView implements InputView {
```

Comment:
Why introduce `AbstractPagedInputView`? I think in our case, we don't have paged `MemroySegment`s as `InputView`. Do not introduce interfaces that we won't use in the near future.